ETC.tsv

```diff
@@ -13345,67 +13345,65 @@ ETC_20150717_013345	button enables you to reset all items registered for Buy and
 ETC_20150717_013346	{img mouseclick_right 40 40} on the item from 'Items Sold' below to buy back item delete it from the list.
 ETC_20150717_013347	You need the Anvil and money to Enhance equipments.
 ETC_20150717_013348	can be purchased form Item Merchant or obtained from monsters.
-ETC_20150717_013349	When you {img mouseclick_right 40 40} on the Anvil and click the equipment, Enhancement window will appear. Select
+ETC_20150717_013349	Press {img mouseclick_right 40 40} on the Anvil, then select an equipment item to open the Enhancement window. Select
 ETC_20150717_013350	and
 ETC_20150717_013351	will appear. Hit the
 ETC_20150717_013352	three times to Enhance the item.
-ETC_20150717_013353	If enhancement of weapons above level +4 fails, its Potential will be consumed. When the Potential falls below 0, the item will be destroyed.
-ETC_20150717_013354	For armors, bracelets, and necklaces, enhancement penalty applies above +2.
+ETC_20150717_013353	When enhancing +4 weapons and above, Potential will be consumed on failure. The item will be destroyed when Potential falls below zero.
```

Comment:
When enhancing +4 weapons and above, Potential will be consumed on failure. The item will be destroyed when Potential falls below zero. **->** 

1. When enhancing a weapon to +5 or higher, Potential will be consumed and the enhancement effect will be reset upon failure. The item will be destroyed when the Potential reaches 0. 
2. Potential will be consumed and the enhancement effect will reset when you failed to enhance a weapon to +5 or higher. The item will be destroyed when the Potential reaches 0. 

Pick whatever one is better.

The original text says +5 instead of +4. In addition, it says the item will be destroyed when Potential hits 0. Not when it falls below 0.